frontend/packages/container-security/src/components/image-manifest-vuln.tsx

```diff
@@ -231,15 +231,19 @@ export const ImageManifestVulnTableRow: RowFunction<ImageManifestVuln> = ({
         <ResourceLink kind="Namespace" name={namespace} />
       </TableData>
       <TableData className={tableColumnClasses[2]}>
-        <SecurityIcon
-          color={vulnPriority.find(({ title }) => obj.status.highestSeverity === title).color.value}
-        />
-        &nbsp;{obj.status.highestSeverity}
+        {_.get(obj.status, 'highestSeverity') ? (
+          <>
+            <SecurityIcon color={priorityFor(_.get(obj.status, 'highestSeverity')).color.value} />
```

Comment:
This `_.get` shouldn't be needed since we already check for `highSeverity` above.

```suggestion
            <SecurityIcon color={priorityFor(obj.status.highestSeverity).color.value} />
```